simple-mpc-vars.el

```diff
@@ -53,6 +53,23 @@ with `simple-mpc-seek-forward' and `simple-mpc-seek-backward'."
   :group 'simple-mpc
   :type 'integer)
 
+(defcustom simple-mpc-playlist-auto-refresh nil
+  "Whether the playlist should be automatically refreshed every
+`simple-mpc-playlist-auto-refresh-interval' seconds.
+
+This is useful if you are listening to online radio."
+  :group 'simple-mpc
+  :type 'boolean)
+
```

Comment:
Could we maybe save the refresh interval directly in this variable? Then we can assume a value of nil (or 0) means to disable the functionality and anything non-nil meaning the functionality should be enabled. Might be a bit easier for users to configure? What do you think?